ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -37,6 +37,11 @@ interface {{ ethernet_interface.name }}
 {%     elif ethernet_interface.logging.event.link_status is arista.avd.defined(false) %}
    no logging event link-status
 {%     endif %}
+{%     if ethernet_interface.logging.event.congestion_drops is arista.avd.defined(true) %}
```

Comment:
Please wrap the two logging.event checks in an outer `if ethernet_interface.logging.event is arista.avd.defined` so we save a little processing for 90% of the deployments.

Reply:
Addressed in f8117fa